src/Type/PostObject/Mutation/PostObjectMutation.php

```diff
@@ -95,6 +97,59 @@ public static function input_fields( $post_type_object ) {
 				],
 			];
 
+			/**
+			 * Add inputs for connected taxonomies
+			 */
+			$allowed_taxonomies = \WPGraphQL::$allowed_taxonomies;
+			if ( ! empty( $allowed_taxonomies ) && is_array( $allowed_taxonomies ) ) {
+				foreach ( $allowed_taxonomies as $taxonomy ) {
+					// If the taxonomy is in the array of taxonomies registered to the post_type
+					if ( in_array( $taxonomy, get_object_taxonomies( $post_type_object->name ), true ) ) {
+						$tax_object                                       = get_taxonomy( $taxonomy );
+
+						$node_input = new WPInputObjectType( [
+							'name'   => $post_type_object->graphql_single_name . ucfirst( $tax_object->graphql_plural_name ) . 'Nodes',
+							'description' => sprintf( __( 'List of %1$s to connect the %2$s to. If an ID is set, it will be used to create the connection. If not, it will look for a slug. If neither are valid existing terms, and the site is configured to allow terms to be created during post mutations, a term will be created using the Name if it exists in the input, then fallback to the slug if it exists.', 'wp-graphql' ), $tax_object->graphql_plural_name, $post_type_object->graphql_single_name ),
+							'fields' => [
+								'id'   => [
+									'type'        => Types::id(),
+									'description' => sprintf( __( 'The ID of the %1$s. If present, this will be used to connect to the %2$s. If no existing %1$s exists with this ID, no connection will be made.', 'wp-graphql' ), $tax_object->graphql_single_name, $post_type_object->graphql_single_name ),
+								],
+								'slug' => [
+									'type'        => Types::string(),
+									'description' => sprintf( __( 'The slug of the %1$s. If no ID is present, this field will be used to make a connection. If no existing term exists with this slug, this field will be used as a fallback to the Name field when creating a new term to connect to, if term creation is enabled as a nested mutation.', 'wp-graphql' ), $tax_object->graphql_single_name ),
+								],
+								'description'   => [
+									'type'        => Types::id(),
```

Comment:
Shouldn't this be the `string` type? Also, maybe add this field to your tests.

Reply:
Never-mind about the tests, it looks like your tests cover those lines and they're passing. 🤔

Reply:
yes, it should be a string. 

The ID type is flexible and can be a string or integer, but this _should_ be a string